flight/PiOS/Common/pios_hal.c

```diff
@@ -220,12 +220,12 @@ void PIOS_HAL_ConfigureCom(const struct pios_usart_cfg *usart_port_cfg,
  * @param[in] usart_dsm_cfg Configuration for the USART for DSM mode.
  * @param[in] dsm_cfg Configuration for DSM on this target
  * @param[in] usart_com_driver The COM driver for this USART
- * @param[in] bind A number of binding pulses to transmit on this port.
+ * @param[in] mode Mode in which to operate DSM driver; encapsulates binding
  */
 static void PIOS_HAL_ConfigureDSM(const struct pios_usart_cfg *usart_dsm_cfg,
                 const struct pios_dsm_cfg *dsm_cfg,
                 const struct pios_com_driver *usart_com_driver,
-		int bind)
+		uint8_t mode)
```

Comment:
should also be the hwshared enum type.  Also I notice this part of the file is already indented wrong and needs a reindent in the future.